runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -92,6 +105,135 @@ MM_StandardAccessBarrier::tearDown(MM_EnvironmentBase *env)
 	MM_ObjectAccessBarrier::tearDown(env);
 }
 
+/**
+ * Unmarked, heap reference, about to be deleted (or overwritten), while marking
+ * is in progress is to be remembered for later marking and scanning.
+ */
+void
+MM_StandardAccessBarrier::rememberObject(MM_EnvironmentBase *env, J9Object *object)
```

Comment:
we will now have 2 completely different remember actions in barrierr world:
- as part of a generational barrier, we need to remember an old object (referencing a new object); the action could be more specifically called remember-object-as-root (in subsequent GCs)
- as part SATB concurrent marking, we need to remember old reference point being overwritten; the action could be more specifically called remember-object-to-rescan (in this GC cycle)

Therefore, the name of methods should be appropriately renamed to avoid confusion. Renaming this method should be sufficent, for now. Creative suggestions are welcome.

Reply:
renamed the method to rememberObjectToRescan